metldata/load/event_publisher.py

```diff
@@ -164,3 +180,13 @@ async def process_resource_upsert(self, *, resource: SearchableResource):
             key=f"dataset_embedded_{resource.accession}",
             topic=self._config.resource_change_event_topic,
         )
+
+    def is_primary_dataset_source(
+        self, *, artifact_name: str, resource_class_name: str
+    ) -> bool:
+        """Checks if combination of artifact name and resource class name describe the
+        configured source for outbound change events"""
```

Comment:
```suggestion
        configured source for outbound dataset change events"""
```

Reply:
Changed in b1142b1c72c3b538fb6e712b3ae4b583fe386e0f